kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/ContentHostingComparator.java

```diff
@@ -94,6 +94,10 @@ public ContentHostingComparator(String property, boolean ascending, boolean is_s
 	 */
 	public int compare(Object o1, Object o2)
 	{
+		if (o1 == null && o2 == null) { return 0; }
```

Comment:
I assume no, but I wasn't sure. The original function was doing null checks in the middle of the function, so I moved it to the top in case they are actually needed.